tests/tensorflow/sparsity/rb/test_integration.py

```diff
@@ -137,10 +137,10 @@ def test_rb_sparse_target_lenet(distributed, deterministic_mode):
 
     batch_size = 128
     if distributed:
-        num_of_replicas = 3
-        strategy = tf.distribute.MirroredStrategy([f"GPU:{i}" for i in range(num_of_replicas)])
+        coeff = 3
+        strategy = tf.distribute.MirroredStrategy()
```

Comment:
There are only two GPUs in our environment. This is the reason why our jobs are stuck.